frontend/src/layout.ts

```diff
@@ -71,4 +70,4 @@ export class StatefulLayout {
   getNodeColor() {
     return this.colors
   }
-}
+}
```

Comment:
new line

Reply:
Not sure I understand what you mean here 😅

Reply:
files should always end with an empty new line, github shows a little cross symbol if it is not there.